.github/workflows/cli-release.yml

```diff
@@ -0,0 +1,83 @@
+name: CLI Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      draft-release:
+        description: 'Draft release?'
+        required: false
+        type: boolean
+        default: true
+      prerelease:
+        description: 'Prerelease?'
+        required: false
+        type: boolean
+        default: true
+      npm-version:
+        description: 'NPM version bump?'
+        required: false
+        type: choice
+        options: 
+          - major
+          - minor
+          - patch
```

Comment:
This choice input is neat, I'm stealing this for the other release workflows 😄

Reply:
😂👍🏻